### DISCUSSION TITLE
Hosted control planes for Talos

### DISCUSSION THREAD

**Discussion by shanduur** [siderolabs/talos#8926] [Ideas]
Hosted Control Planes (HCP) is an innovative approach designed to streamline the deployment and management of Kubernetes clusters, ensuring high availability, multi-tenancy, and resilience. A system akin to Kamaji but tailored for Talos, providing an optimized experience for managing multiple Kubernetes clusters. This is something that is already utilized e.g. in the E2E tests, so this might require nothing more than providing Helm Chart/Kustomization for the control plane and documentation.

**Comment by Anutrix** [comment]
Sad that this is blocking Cozystack from using Talos's Kubernetes Cluster.

**Comment by s04** [comment]
Bump! :

**Comment by rothgar** [comment]
This is being worked on in https://github.com/siderolabs/omni/issues/549
